tests/CMakeLists.txt

```diff
@@ -36,6 +36,10 @@ ADD_EXECUTABLE(unittest_sink nnstreamer_sink/unittest_sink.cpp ${gtestSrc})
 TARGET_INCLUDE_DIRECTORIES(unittest_sink PUBLIC ${pkgs_INCLUDE_DIRS} ${testpkgs_INCLUDE_DIRS} ${gtestInc})
 TARGET_LINK_LIBRARIES(unittest_sink ${pkgs_LIBRARIES} ${testpkgs_LIBRARIES} ${gtestLink})
 
+# Unit test uses this custom filter
+ADD_LIBRARY(nnscustom_framecounter SHARED nnstreamer_sink/nnscustom_framecounter.c)
+TARGET_LINK_LIBRARIES(nnscustom_framecounter nnstreamer)
```

Comment:
nnstreamer defined only SINGLE_BINARY=ON (use ${pkgs_LIBRARIES} instead?)

Reply:
Anyway, pkg_LIBRARIES won't be appropraite for custom filters. They only need ```libtensor_filter.so```.
Plus, with SINGLE_BINARY=OFF, other custom filters are failing to build :)
I'll fix it with another commit

Reply:
Refer to the third commit in this PR.